mac

```diff
@@ -166,29 +166,62 @@ brew_install_or_upgrade 'hub'
 # shellcheck disable=SC2016
 append_to_file "$HOME/.zshrc" 'eval "$(hub alias -s)"'
 
+
+fancy_echo 'Checking on Node.js installation...'
+
 if ! brew_is_installed "node"; then
-  if ! command -v n > /dev/null; then
-    fancy_echo 'Installing n and latest Node.js and NPM...'
-    curl -L http://git.io/n-install | bash -s -- -y latest
+  if command -v nvm > /dev/null; then
```

Comment:
Heh. I wasn't thinking clearly when I first saw this. The echoed statement in the else clause threw me off because it implied that nvm was installed, but the if statement was checking for the presence of nvm instead of the absence. Adding `!` here fixes all the things.